include/nuttx/clock.h

```diff
@@ -436,6 +436,22 @@ int clock_systime_timespec(FAR struct timespec *ts);
 int clock_cpuload(int pid, FAR struct cpuload_s *cpuload);
 #endif
 
+/****************************************************************************
+ * Name:  clock_systime_steps
+ *
+ * Description:
+ *   Add system time by idle ticks.
+ *
+ * Input Parameters:
+ *   ticks - Idle ticks
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+void clock_systime_steps(sclock_t ticks);
```

Comment:
should we move it to arch.h like nxsched_process_timer?

Reply:
I think from sched level, OS don't need to know whether this wakeup come from the deep sleep or not. Why not reuse nxsched_process_timer instead?